multiplatform-movies/android/src/main/java/com/novoda/movies/gallery/GalleryActivity.kt

```diff
@@ -2,16 +2,26 @@ package com.novoda.movies.gallery
 
 import android.os.Bundle
 import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.GridLayoutManager
+import android.widget.Toast
 import com.novoda.movies.R
 import kotlinx.android.synthetic.main.activity_gallery.*
 
+private const val NO_OF_COLUMNS = 3
```

Comment:
`NO` or `NB`?

Reply:
`No.` Seems to be the correct abbreviation for number : 
https://english.stackexchange.com/questions/43095/what-is-the-correct-abbreviation-for-the-word-numbers

Reply:
Interesting, always thought it was NB, maybe in french?
👍